.github/workflows/application-test-scenario.yaml

```diff
@@ -0,0 +1,46 @@
+name: Application Specific Test Scenario
+on:
+  workflow_dispatch:
+    inputs:
+      scenario:
+        description: 'The scenario to run, e.g., install or upgrade'
+        required: true
+        default: 'install'
+        type: string
+      app:
+        description: 'The kommander application to test, e.g. reloader,kube-prometheus-stack'
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      scenario:
+        description: 'The scenario to run, e.g., install or upgrade'
+        required: true
+        type: string
+      app:
+        description: 'The kommander application to test, e.g. reloader,kube-prometheus-stack'
+        required: true
+        type: string
+
```

Comment:
```suggestion

env:
  UPGRADE_KAPPS_REPO_PATH: '.work/upgrade/kommander-applications'
  UPGRADE_FROM_KOMMANDER_VERSION: 'v2.7.0'

```

Reply:
Do you mind adding this?

Reply:
Or even better:

```
$ gh dkp generate upgrade-matrix --json | jq --raw-output ".upgrades | to_entries | [last][0].value.from"
v2.8.0-dev
```

Reply:
Actually, we need to upgrade from multiple version, e.g, right now 2.7 branch has to be tested with the following upgrade paths:

```
$ gh dkp generate upgrade-matrix                                                                   
From version | To version | k8s Version
v2.6.0       | v2.7.1-dev | 1.26
v2.6.1       | v2.7.1-dev | 1.26
v2.6.2       | v2.7.1-dev | 1.26
v2.6.3-dev   | v2.7.1-dev | 1.26
v2.7.0       | v2.7.1-dev | 1.27
```

The following `gh-dkp` commands might give you the information necessary to compute this:

```
gh dkp generate dev-version
gh dkp generate dev-versions
gh dkp generate upgrade-matrix
```

They all have json output.

Reply:
@mikolajb let's address this in another PR